ArduCopter/Parameters.cpp

```diff
@@ -133,7 +133,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: RTL_CONE_SLOPE
     // @DisplayName: RTL cone slope
     // @Description: Defines a cone above home which determines maximum climb
-    // @Range: 0.5 10.0
+    // @Range: 0.0 10.0
```

Comment:
This is an interesting one, I think there might be others like it. Basically we want range to be 0.5 to 10.0, but with 0 being a special disable value. I'm not sure how we can fix this, but changing the range is incorrect, as between 0.0 and 0.5 it won't do anything.